controllers/semeru_compiler.go

```diff
@@ -549,8 +549,8 @@ func (r *ReconcileOpenLiberty) getSemeruJavaOptions(instance *openlibertyv1.Open
 		args := []string{
 			"/bin/bash",
 			"-c",
-			"export OPENJ9_JAVA_OPTIONS=\"$OPENJ9_JAVA_OPTIONS " +
-				jitSeverOptions +
+			"export OPENJ9_JAVA_OPTIONS=\"$OPENJ9_JAVA_OPTIONS " + jitSeverOptions +
+				"\" && export OPENJ9_RESTORE_JAVA_OPTIONS=\"$OPENJ9_RESTORE_JAVA_OPTIONS " + jitSeverOptions +
```

Comment:
@NottyCode points out that this handles `OPENJ9_JAVA_OPTIONS` and `OPENJ9_RESTORE_JAVA_OPTIONS` values being set in kubernetes config but it doesn't handle cases where the container image has already set the env.  For example, using `ENV` in their `Dockerfile`.  Is this a concern?  If so it has been an issue ever since the operator started auto setting the `OPENJ9_JAVA_OPTIONS`

Reply:
It's not a concern. The commands will be executed at runtime inside the container. So the _jitServerOptions_ will be appended to `OPENJ9_JAVA_OPTIONS` value (the value could have come from container ENV or k8s env). In fact, we took this approach in the first place to preserve the `OPENJ9_JAVA_OPTIONS` ENV specified in the Liberty container image:

```
ENV RANDFILE=/tmp/.rnd \
    OPENJ9_JAVA_OPTIONS="-XX:+IgnoreUnrecognizedVMOptions -XX:+IdleTuningGcOnIdle -Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal -Dosgi.checkConfiguration=false"
```